src/fromager/context.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         # be built in order.
         self._build_stack: list[typing.Any] = []
         self._build_requirements: set[tuple[NormalizedName, str]] = set()
+        self.all_edges: dict[NormalizedName, tuple(str, NormalizedName, Version)] = {}
```

Comment:
```suggestion
        self.all_edges: dict[NormalizedName, list[tuple(str, NormalizedName, Version)]] = {}
```
Looking at `update_dependency_graph` method, the type of value seems to be a list

Reply:
Good catch, thanks!